Added a Slack Notification section in the cli init doc (#790)
great-expectations/great_expectations: docs/getting_started/cli_init.rst

Configuring Slack Notifications
----------------------------------------

> Great Expectations can post messages to a Slack channel when a new dataset is validated. This helps teams to monitor data quality in their pipeline in real time. Here is what these messages look like:

Reviewer: ```suggestion
Great Expectations can post messages to a Slack channel each time a dataset is validated.
```
